assets/js/googlesitekit/notifications/datastore/notifications.js

```diff
@@ -359,6 +437,30 @@ export const reducer = createReducer( ( state, { type, payload } ) => {
 			break;
 		}
 
+		case MARK_NOTIFICATION_SEEN: {
+			const { notificationID, currentDate } = payload;
+			const seenNotifications = { ...state.seenNotifications };
+
+			// Initialize array if it doesn't exist
```

Comment:
Comments should probably be full sentences. 😅 

```suggestion
			// Initialize array if it doesn't exist.
```